core/src/main/scala/org/apache/spark/storage/BlockManagerDecommissioner.scala

```diff
@@ -125,7 +126,10 @@ private[storage] class BlockManagerDecommissioner(
                   logDebug(s"Migrated sub-block $blockId")
                 }
               }
-              logInfo(s"Migrated $shuffleBlockInfo to $peer")
+              lazy val endTime = System.currentTimeMillis()
+              lazy val totalBlockSize = Utils.bytesToString(blocks.map(b => b._2.size()).sum)
+              logInfo(s"Migrated $shuffleBlockInfo (" +
+                s"size: $totalBlockSize) to $peer in ${endTime - startTime} ms")
```

Comment:
How about
```suggestion
              logInfo(s"Migrated $shuffleBlockInfo (" +
                s"size: ${Utils.bytesToString(blocks.map(b => b._2.size()).sum)}) to $peer in ${System.currentTimeMillis() - startTime} ms")
```
to avoid those two lazy variables?

Reply:
Or we can simply add an if condition to check if info is enabled :-)

Reply:
`logInfo` already does that inside. @mridulm

Reply:
Yes, but you can avoid all the `lazy val` code.